REFERENCE.md

```diff
@@ -199,6 +200,26 @@ The cross-validation procedure works as follows:
   - While the notion of error is straightforward for regression forests, it can be more subtle in the context of treatment effect estimation. For causal forests, we use a measure of error developed in Nie and Wager (2017) motivated by residual-on-residual regression (Robinson, 1988).
 - Finally, given the debiased error estimates for each set of parameters, we apply a smoothing function to determine the optimal parameter values.
 
+
+### Boosting Random Forests
+
+**Note:** this feature is currently marked 'experimental'. Its implementation may change in future releases.
+
+Ghosal and Hooker (2018), show how a boosting step can reduce bias in random forest predictions. We have included a  `boosted_regression_forest` feature, which trains a series of forests, each of which are trained on the OOB residuals from the previous step. `boosted_regression_forest` includes the same parameters as `regression_forest`, which are passed directly to the forest trained in each step. It also includes control over the step selection procedure:
+
+- If `boost.steps` is not specified, then a cross-validation procedure selects the number of boosting steps automatically, where the next step is only taken if the estimated OOB error for the next step, computed using a small forest of size `boost.trees.tune`, has an estimated OOB error reduction from  the previous step of more than `boost.error.reduction`. For computational considerations only up to `boost.max.steps` are considered.
```

Comment:
I think this may be too concise to be easily understandable to someone new to the algorithm. A suggestion for how to restructure this section (starting from the phrase 'It also includes control over the step selection procedure:'):

"By default, the number of boosting steps is automatically chosen through a cross-validation procedure. This procedure works as follows:
- First, a single `regression_forest` is trained.
- To decide if another step should be taken, we train a small forest ...
- If another step should be taken, then we train a full-sized `regression_forest` on the residuals and add it to the boosted forest.
- This process continues until `boost.error.reduction` cannot be met when training the mini-forest, or if the total number of steps exceeds the limit `boost.max.steps`.

If you would like to skip the cross-validation procedure and specify the number of steps directly, then you can pass the parameter `boost.steps`."

Also, can we add a sentence discussing how `tune.parameters` is handled with boosting?

Reply:
thanks, this is much better